src/processor.rs

```diff
@@ -1,37 +1,81 @@
-use anyhow::Result;
+use std::sync::Arc;
+
+use actix_web::{http::Method, web, HttpRequest, HttpResponse, HttpResponseBuilder};
+use anyhow::{bail, Result};
 // pyO3 module
-use crate::types::PyFunction;
+use crate::types::{Headers, PyFunction};
+use futures_util::stream::StreamExt;
 use pyo3::prelude::*;
 
-use hyper::{Body, Response, StatusCode};
+/// @TODO make configurable
+const MAX_SIZE: usize = 10_000;
```

Comment:
What is the MAX_SIZE referring to?
This size of the POST request?

Reply:
Yeah the biggest body payload